MekHQ/src/mekhq/campaign/mission/enums/MissionStatus.java

```diff
@@ -25,24 +25,32 @@
 
 public enum MissionStatus {
     //region Enum Declarations
-    ACTIVE("MissionStatus.ACTIVE.text"),
-    SUCCESS("MissionStatus.SUCCESS.text"),
-    FAILED("MissionStatus.FAILED.text"),
-    BREACH("MissionStatus.BREACH.text");
+    ACTIVE("MissionStatus.ACTIVE.text", "MissionStatus.ACTIVE.toolTipText"),
```

Comment:
Gotta say, I really like the idea of being able to associate arbitrary metadata with enum values.

Reply:
Certainly wish it was available in other languages! First class enums have a lot of advantages.